### PR TITLE
doc: add cases when ceph-extras is not needed

### PR DESCRIPTION
**Pull request by trhoden**
The Ceph Extras repo is not needed on EL7 distributions
or Fedora.

Signed-off-by: Travis Rhoden <trhoden@redhat.com>

### CODE REVIEW

**Comment by trhoden** [comment]
@ktdreyer Can you make sure what I've said here is accurate?

Also, for the life of me I cannot see why it thinks the last line in the file(s) are different.  :/

**Comment by ktdreyer** [comment]
The text looks fine to me! Thanks for doing this.

You can add a message to the commit log in order to indicate that this is associated with tracker.ceph.com/issues/9793 . Add the following paragraph:

`http://tracker.ceph.com/issues/9547 Refs: #9547`

If you grep `git log` you can see we use this convention in a couple other commits. (Well, "we" meaning Loic and myself :) But I think it's a good convention.

Regarding the changes to the final lines of the two files, GitHub's UI doesn't make it immediately obvious what's changed, so if you add `.patch` onto the URL of this pull request you'll get a plaintext patch of your change, and this will show you what's going on: https://github.com/ceph/ceph/pull/3339.patch

Prior to your commit, the text files did not have newlines on the final lines. Whoever wrote these files was using an editor that did not append a final newline character. It's likely your editor added the newline characters when you hit "save" - I know Vim does this a lot for me. Some of the ways that I've fixed this in the past:

1. If I know a file happens to be in this condition before I make my desired change, I will open it in Vim, and before saving it, I'll run `:set noeol` so that Vim will preserve the lack of a newline character. I can then commit my change as usual and the lack of the newline will be preserved.
2. If I didn't realize it prior to saving, and I already overwrote the file in Vim with the extra newline, then I'll use `git add -i` to interactively stage my desired changes, then use `git commit` to only check in the changes without the newlines. Then I can use `git reset --hard` to undo the newline alterations that were not staged/committed.
3. Let's say I didn't get a chance to run `git add -i` - not only did I overwrite the newlines in the file, but I committed them as well (like what's done here). This happens to me a lot :) In that case, I'll use [this Perl snippet](http://stackoverflow.com/questions/8635240/remove-newline-from-end-of-file) to trim the newline change and then `git commit -a --amend` to redo the commit without the newline change.

Hope that helps!

File: doc/install/install-vm-cloud.rst [hunk 1]
```diff
@@ -57,9 +57,13 @@ To install QEMU, execute the following:
...
 	enabled = 1
 
+.. note:: ceph-extras is not needed for EL-7 variants and Fedora.  It is necessary
+   for EL-6.
+
 #. Create a ``/etc/yum.repos.d/ceph-extras.repo`` file with the following 
-   contents, and replace ``{distro}`` with your Linux distribution. Follow
-   the ``baseurl`` path below to see which distributions Ceph supports:: 
+   contents, and replace ``{distro}`` with your Linux distribution. Only
+   older distributions require ceph-extras. Follow the ``baseurl`` path
```
Comment (alfredodeza) [inline doc/install/install-vm-cloud.rst]: What is an 'old distribution' ? As in: older than what?

Comment (trhoden) [inline doc/install/install-vm-cloud.rst]: Good point.  Would you be happier if I just dropped that line entirely?  I already define what versions require/don't require ceph-extras in the ..note immediately before this.

Or I can just say something like "Only -EL6 based distributions and older...."

If I do the latter, is the "and older" bit necessary?  I don't actually know if we can install on EL5.  I don't think we can.  So maybe it is better to just say "Only -EL6 based distributions require ceph-extras"

**Comment by trhoden** [comment]
Making this a bit more forward-looking, would it be better instead to say:

`ceph-extras on RPM-based systems is only needed on EL6-based distributions (RHEL, CentOS, Scientific).  It is not needed for Fedora or as of EL7.`

Something like that, @ktdreyer and @alfredodeza ?

**Comment by ktdreyer** [comment]
How about this:

`ceph-extras on RPM-based systems is only needed on EL6-based distributions (RHEL 6, CentOS 6, Scientific Linux 6). It is not needed for Fedora or RHEL 7+.`

**Comment by alfredodeza** [comment]
@ktdreyer 's suggestion sounds good to me

**Pull request closed without merging**